index.bs

```diff
@@ -238,16 +249,20 @@ restrictions it asserts. Likewise, user agents MUST also take into account the e
 asserted by the document's opener or embedder, ensuring that they're properly imposed as well. To do
 so, HTML is patched as follows:
 
-1.  An <dfn>embedder policy</dfn> is a string with one of the following values: "`unsafe-none`",
-    "`require-corp`".
+1.  An <dfn>embedder policy</dfn> consists of:
+
+    1.  A string (<dfn for="embedder policy">value</dfn>) with one of the following values:
+        "`unsafe-none`", "`require-corp`", initially "`unsafe-none`".
+
+    1.  A nullable string (<dfn for="embedder plicy">reporting endpoint</dfn>), initially null.
```

Comment:
```suggestion
    1.  A nullable string (<dfn for="embedder policy">reporting endpoint</dfn>), initially null.
```

Reply:
Elsewhere we describe this as "A string or `null`". I'm not sure "nullable string" is a concept HTML knows about (it doesn't appear in a quick search). @annevk might have opinions about whether it should. :)

Reply:
I'd prefer sticking with a string or null.

Reply:
Done.